### PR TITLE
HPCC-13507 Translations

### PR DESCRIPTION
**Pull request by GordonSmith**
BS, HR + SR

Fixes HPCC-13507
Fixes HPCC-13506
Fixes HPCC-13505

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@emuharemagic Please Review
@miguelvazq Please Review

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success
HPCC Start: OK
HPCC Stop: OK

File: esp/src/eclwatch/nls/bs/hpcc.js [hunk 1]
```diff
@@ -59,11 +60,14 @@
...
     Comment: "Komentar",
     Completed: "Kompletiran",
+    ComplexityWarning: "Više od praga {threshold} aktivnosti ({activityCount}) - zaustavite prikaz podataka?'",
     Component: "Komponenta",
     Compress: "Sabijte",
     Compressed: "Komprimirani",
+    CompressedFileSize: "Komprimirana Veličina Datoteke",
     Configuration: "Konfiguracija",
     ConfirmPassword: "Potvrdite Lozinku",
+    ConfirmRemoval: "Jeste li sigurni da to želite učiniti ?",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/bs/hpcc.js]: Extra space after "učiniti"

File: esp/src/eclwatch/nls/bs/hpcc.js [hunk 2]
```diff
@@ -59,11 +60,14 @@
     Command: "Komanda",
     Comment: "Komentar",
     Completed: "Kompletiran",
+    ComplexityWarning: "Više od praga {threshold} aktivnosti ({activityCount}) - zaustavite prikaz podataka?'",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/bs/hpcc.js]: after "?" there is a apostrophe is this intentional.

File: esp/src/eclwatch/nls/hr/hpcc.js [hunk 3]
```diff
@@ -59,11 +60,14 @@
     Command: "Komanda",
     Comment: "Komentar",
     Completed: "Kompletiran",
+    ComplexityWarning: "Više od praga {threshold} aktivnosti ({activityCount}) - prekinite prikaz podataka?'",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/hr/hpcc.js]: After "?" there is a apostrophe is this intentional?

File: esp/src/eclwatch/nls/hr/hpcc.js [hunk 4]
```diff
@@ -59,11 +60,14 @@
...
     Component: "Komponenta",
     Compress: "Sabijte",
     Compressed: "Komprimirani",
+    CompressedFileSize: "Komprimirana Veličina Datoteke",
     Configuration: "Konfiguracija",
     ConfirmPassword: "Potvrdite Lozinku",
+    ConfirmRemoval: "Jeste li sigurni da to želite učiniti ?",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/hr/hpcc.js]: Extra space after "učiniti"

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 5]
```diff
@@ -60,11 +61,14 @@
...
     Comment: "Коментар",
     Completed: "Комплетиран",
+    ComplexityWarning: "Више од прага {threshold} активности ({activityCount}) - прекините приказ података?",
     Component: "Компонента",
     Compress: "Сабијте",
     Compressed: "Компримирани",
+    CompressedFileSize: "Компримирана Величина Датотеке",
     Configuration: "Конфигурација",
     ConfirmPassword: "Потврдите Лозинку",
+    ConfirmRemoval: "Јесте ли сигурни да то желите учинити ?",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/sr/hpcc.js]: Extra space before ? Intentional?

**Comment by miguelvazq** [comment]
All else looks good.

**Comment by richardkchapman** [comment]
@GordonSmith Please comment

**Comment by GordonSmith** [comment]
@emuharemagic Can you let me know if any changes need to be made based on Miguels comments?

File: esp/src/eclwatch/nls/bs/hpcc.js (hunk 2 above)
Comment (emuharemagic) [inline esp/src/eclwatch/nls/bs/hpcc.js]: It needs to be removed

File: esp/src/eclwatch/nls/bs/hpcc.js (hunk 1 above)
Comment (emuharemagic) [inline esp/src/eclwatch/nls/bs/hpcc.js]: It needs to be removed

File: esp/src/eclwatch/nls/hr/hpcc.js (hunk 3 above)
Comment (emuharemagic) [inline esp/src/eclwatch/nls/hr/hpcc.js]: It needs to be removed

File: esp/src/eclwatch/nls/hr/hpcc.js (hunk 4 above)
Comment (emuharemagic) [inline esp/src/eclwatch/nls/hr/hpcc.js]: It needs to be removed

File: esp/src/eclwatch/nls/sr/hpcc.js (hunk 5 above)
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: It needs to be removed

**Comment by GordonSmith** [comment]
Changes made as per comments.

**Pull request merged**